src/Controls/src/Core/Compatibility/Handlers/ListView/Windows/ListViewRenderer.cs

```diff
@@ -802,6 +802,9 @@ void OnListItemClicked(object sender, ItemClickEventArgs e)
 
 		void OnControlSelectionChanged(object sender, WSelectionChangedEventArgs e)
```

Comment:
Did we validate if this subscription needs to be removed?
https://github.com/dotnet/maui/blob/d6cacfbe9087743b89af19b247a3475843819ebd/src/Controls/src/Core/Compatibility/Handlers/ListView/Windows/ListViewRenderer.cs#L116

I'm still wondering if the handler is getting disconnected and we're not disconnecting all the proper events